packages/radix-vue/src/DateField/DateFieldRoot.vue

```diff
@@ -1,25 +1,23 @@
 <script lang="ts">
-import { type DateValue } from '@internationalized/date'
+import { type DateValue, isEqualDay } from '@internationalized/date'
 
 import type { Ref } from 'vue'
 import type { PrimitiveProps } from '@/Primitive'
 import { type Formatter, createContext, useDateFormatter, useKbd } from '@/shared'
 import {
-  type AnyExceptLiteral,
   type Granularity,
   type HourCycle,
   type Matcher,
   type SegmentPart,
   type SegmentValueObj,
-  type SupportedLocale,
   getDefaultDate,
   hasTime,
   isBefore,
 } from '@/shared/date'
 import { createContent, initializeSegmentValues, isSegmentNavigationKey, syncSegmentValues } from './utils'
 
 type DateFieldRootContext = {
-  locale: Ref<SupportedLocale>
+  locale: Ref<string>
```

Comment:
@epr3  why change from the `SupportedLocale` type to `string` ya? This loses the ability to autcomplete the available locale right?

Reply:
@zernonia The `locale` prop is only used by the `DateFormatter` which is just a wrapper with some polyfills over [`Intl.DateTimeFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat), so just using `SupportedLocale` would not cover all the possible values that are accepted. e.g. `fa-IR`, `de-AT`, `de-DE`, `zh-Hans-CN`

`SupportedLocale` is used for the `DateField` type components to generate localized strings for the placeholders like `YYYY/MM/DD`.

Reply:
Ahh ic! Thanks for the details!